ADA/ada.qmd

```diff
@@ -19,57 +17,194 @@ It brings together:
+
+You will need to be given access to Databricks and the data you require by the ADA team by completing an "Analytical Data Access Request" service request form in the IT Help Centre. At the moment, not all of the data currently available in SSMS has been migrated to the Delta Lake for use in Databricks. You can [check the status of various datasets](https://app.powerbi.com/groups/me/reports/966389db-2e4e-42b6-8416-36aee2fb9503/ReportSection?ctid=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9&experience=power-bi) in the data catalogue dashboard. 
+
+To access your data and make use of cloud computing resource, you will need access to a cluster, which is a group of virtual machines that complete your processing for you. Your cluster setup will depend on which programming language you wish to use. You will be given access to a shared cluster when your data access is granted. Please see the [how clusters work](https://analytical-data-access.education.gov.uk/get-support/how-clusters-work) page on the ADA guidance site for more detail. 
+
+---
+
+#### Writing and running code
+
+---
+
+In the Databricks interface, you have the choice to use notebooks, scripts or the Databricks SQL Editor. 
+
+Notebooks are made up of multiple code or markdown cells. They are different to IDEs (Integrated Development Environments) like RStudio. You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks).
+
+Scripts (e.g. R scripts) can be created and saved in your Workspace and run via the Databricks interface. 
+
+There is detailed guidance on using the Databricks SQL editor, which is similar to SSMS, on the [ADA guidance site](https://analytical-data-access.education.gov.uk/get-support/using-sql-in-databricks-sql-editor).
+
+One benefit of Databricks is the use of workflows to run your code. Workflows allow you to automate the running of single or multiple scripts, so if you have large amounts of data processing to do on a regular basis, you can set up a workflow to complete this overnight. Your laptop does not need to be switched on for this to work. 
+
+You can also run code from RStudio or other IDEs locally and execute it against the data held in Databricks. To do this, you will need to set up a connection between RStudio and the Databricks SQL Warehouse. This is explained in the [R scripts in RStudio section](/ADA/ada.html#r-scripts-in-rstudio) below.
+
+---
+
+#### Accessing code from a git repository
+
+Databricks has inbuilt git functionality, and you can save your notebooks and scripts inside git repositories as with any other file formats. There is guidance on using Databricks with Azure DevOps on our [Databricks and version control page](ADA/git_databricks.html). 
+
+---
+
+## What this means for existing code
+
+### SQL scripts
```

Comment:
You could add a sentence beneath this header to spell out that once data is migrated to their databases you won't access / be able to run queries in SMSS as you previously would have done. 
Then the sections below work nicely to cover the scenarios where you might rely on a SQL script

Reply:
I'm not sure this is true @lauraselby? You can still dual run according this other section
![image](https://github.com/user-attachments/assets/323a2c34-8027-438b-a3e1-ae1182df2ac2)

Maybe that important box text comes here?

Reply:
I'm guessing at some point they will be getting rid of SSMS though right (after a dual run period). I was trying o bring it into context for what people know who have less awareness of the actual detail - like that software your teams use to connect to all their data won't be available anymore - but don't feel strongly

Reply:
I get the point you're making @lauraselby, I think I'm happy the latest update covers it enough from our side.

Ultimately it's something that is on ADA themselves to be making really clear as this is the core of their project, so the main messaging on that should be done by them as they'll be the ones seeing it through (and currently there's not enough evidence we're close enough to turning stuff off for us to go any stronger)